ext/openssl/tests/openssl_pkcs12_read_basic.phpt

```diff
@@ -0,0 +1,21 @@
+--TEST--
+openssl_pkcs12_read() tests
+--SKIPIF--
+<?php if (!extension_loaded("openssl")) print "skip"; ?>
+--FILE--
+<?php
+$p12_file = dirname(__FILE__) . "/p12_with_extra_certs.p12";
+$p12 = file_get_contents($p12_file);
+$certs = array();
+$pass = "qwerty";
+
+var_dump(openssl_pkcs12_read("", $certs, ""));
+var_dump(openssl_pkcs12_read($p12, $certs, ""));
+var_dump(openssl_pkcs12_read($p12, $certs, $pass));
+var_dump(count($certs));
```

Comment:
might be also good to test that the expected certs are returned so we verify that it's not just filling an array with some random values.